pkg/boot/systembooter/bootbooter_test.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2021 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package systembooter
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestNewBootBooter(t *testing.T) {
+	type args struct {
+		config []byte
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    Booter
+		wantErr bool
+	}{
+		{
+			name:    "Boot Booter",
+			args:    args{[]byte(`{"type": "boot"}`)},
+			want:    &BootBooter{Type: "boot"},
+			wantErr: false,
+		},
+		{
+			name:    "Error Boot Booter",
+			args:    args{[]byte(`{"type": "pxeboot"}`)},
+			want:    nil,
+			wantErr: true,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got, err := NewBootBooter(tt.args.config)
+			if (err != nil) != tt.wantErr {
```

Comment:
can you change this to use the errors package?

Reply:
Done